sdks/python/apache_beam/runners/worker/sdk_worker_main_test.py

```diff
@@ -174,26 +185,23 @@ def test__set_log_level_overrides(self):
         )
     ]
     for case, expected in test_cases:
-      options_dict = {'sdk_harness_log_level_overrides': case}
-      sdk_worker_main._set_log_level_overrides(options_dict)
+      overrides = self._overrides_case_to_option_dict(case)
+      sdk_worker_main._set_log_level_overrides(overrides)
       for name, level in expected.items():
         self.assertEqual(logging.getLogger(name).getEffectiveLevel(), level)
 
   def test__set_log_level_overrides_error(self):
     test_cases = [
-        ({
```

Comment:
Now illegal json command as argument fails as early as pipeline option parsing. So that these test cases for error handling changed to valid json but unexpected arrays or invalid levels.

Reply:
ack, thanks.